### PR TITLE
added code for metric reporter

### PR DESCRIPTION
**Pull request by g8y3e**

### CODE REVIEW

File: deploy_doc.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -x
```
Comment (ramimoshe) [inline deploy_doc.sh:1]: we dont need it anymore, no?

File: drivers/appoptics.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
+'use strict';
```
Comment (ramimoshe) [inline drivers/appoptics.js:1]: please align with 
async/await
and const instead of let

File: deploy_doc.sh (hunk 1 above)
Comment (g8y3e) [reply deploy_doc.sh:1]: it for doc generator - for gh-pages that will generate doc like this:
https://ironsource.github.io/atom-node/

File: package.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "ds-metric-reporter-node",
+  "version": "1.0.0",
+  "description": "DS-Atom Metric Reporter",
+  "main": "metric_reporter.js",
+  "author": "g8y3e",
+  "license": "MIT",
+  "dependencies": {
+    "request": "2.88.0",
```
Comment (ramimoshe) [inline package.json:9]: request?

Comment (g8y3e) [reply package.json:9]: ? - yeah we sending post request to appoptics

Comment (ramimoshe) [reply package.json:9]: use superagent instead please

Comment (g8y3e) [reply package.json:9]: ok

File: drivers/appoptics.js (hunk 2 above)
Comment (g8y3e) [reply drivers/appoptics.js:1]: done

File: package.json (hunk 3 above)
Comment (g8y3e) [reply package.json]: changed

File: deploy_doc.sh [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+# Small script to auto-deploy docs to gh-pages branch
+set -e # Exit with nonzero exit code if anything fails
+
+SOURCE_BRANCH="master"
+TARGET_BRANCH="gh-pages"
```
Comment (ramimoshe) [inline deploy_doc.sh:6]: what is it?

**Comment by ramimoshe** [comment]
we have a lot of work to align this lib to the latests features
so lets continue

**Pull request merged**